joomla/joomla-cms
# Give protected menu items container its own place independent of heading type.

### Summary of Changes
Admin menu manager currently uses heading menu type flag to set components menu items container. Now we will have another menu type under "System Links" by name "Components Menu Container"

### Testing Instructions
Create a new custom admin menu.
Create a menu item of type System Links > Components Menu Container
Make sure the components container menu item shows all published menu items of type "Main (Protected)"

### Documentation Changes Required
Not sure.
